### PR TITLE
Mention Glimmer DSL for WX in README.md

### PR DESCRIPTION
**Pull request by AndyObtiva**
Hi,

Thank you for reviving wxruby with the wxruby3 project.

I added a mention of Glimmer DSL for WX in the README.

I am hoping Glimmer DSL for WX could take up the burden of adding value on top of wxruby3 through a declarative visual DSL, Convention over Configuration, Smart Defaults, Bidirectional/Unidirectional Data-Binding, Custom Component support to extend Glimmer's vocabulary, Scaffolding, and more features down the line like Native Executable Packaging.

In any case, I hope you would find value in mentioning Glimmer DSL for WX in the README of wxruby3. If you do, but need me to make changes to the PR first, please let me know.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -182,3 +209,7 @@ maintainable solution.
 
 Please double-check the [INSTALL](INSTALL.md) documents, and search issue archives. If 
 this doesn't help, please post your question using GitHub Issues.
+
+### Is there a simpler and more productive way for writing wxRuby3 desktop GUI applications?
```
Comment (mcorino) [inline README.md:213]: Could you make this read:  "Is there another, more declarative way, for writing wxRuby3 desktop GUI applications?

File: README.md [hunk 2]
```diff
@@ -35,6 +35,16 @@ end
 
 ![Hello_World](assets/hello_world.png "Hello World sample")
 
+Alternatively, you could use [Glimmer DSL for WX](https://github.com/AndyObtiva/glimmer-dsl-wx) on top of wxruby3.
```
Comment (mcorino) [inline README.md:38]: Could you move the alternative Glimmer samples to the Glimmer FAQ section? Maybe under a subheading like "Using Glimmer for WX the examples above could be written like:" ?

**Comment by mcorino** [comment]
Hi Andy,

I have no problem mentioning Glimmer in the README.
I would prefer not to mix up the examples though. That seems a bit confusing for potential users.

I have added some comments to the commit.

File: README.md (hunk 1 above)
Comment (AndyObtiva) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (AndyObtiva) [reply README.md]: I removed the Glimmer examples and moved them to a [Coming From wxruby3](https://github.com/AndyObtiva/glimmer-dsl-wx#coming-from-wxruby3) section in the Glimmer DSL for WX README. I mentioned that section the FAQ item about Glimmer DSL for WX.

**Comment by AndyObtiva** [comment]
OK, I handled your comments and replied to them.

**Review by mcorino — APPROVED** [review]

**Pull request merged**